stories/imdb/stories/blueberry.js

```diff
@@ -3,28 +3,21 @@ import React from 'react'
 import { observable } from 'mobx'
```

Comment:
food for thought - this _should_ be identical to the grey vest story, so we should combine the code. We can either factor out the story and import separately, or maybe even add a theme switcher drop down